packages/grafana-data/src/field/fieldDisplay.ts

```diff
@@ -36,6 +38,7 @@ export interface ReduceDataOptions {
 // TODO: use built in variables, same as for data links?
 export const VAR_SERIES_NAME = '__series.name';
 export const VAR_FIELD_NAME = '__field.name';
+export const VAR_FIELD_LABELS = '__field.labels';
 export const VAR_CALC = '__calc';
 export const VAR_CELL_PREFIX = '__cell_'; // consistent with existing table templates
```

Comment:
maybe unrelated... but we should remove this __cell_N syntax since it will explode.

We could add:
```
__data.row[N].fieldA.text
```
to give you access to anything in the response.  DataLinks already use __data though, so that may break them or be inconsistent

Reply:
yea, really need to change this one.